src/model/project/procedure.rs

```diff
@@ -67,13 +68,24 @@ impl Procedure {
             branches.push(raw_branch.as_str().unwrap().to_string());
         }
 
+        let log: Option<String> = match raw_procedure.get("log") {
+            Some(v) => {
+                if !v.is_string() {
+                    return Err(err_msg("Log format is invalid"));
+                }
+                Some(v.as_str().unwrap().to_string())
+            },
+            None => None
+        };
+
```

Comment:
Not a fan of this check. We could change all other checks to use match or change this to if statements like the others.

Reply:
I did this one this way specifically so that logging could be disabled by removing the line, which is not a feature implemented for any of the other settings as far as I can tell?

Reply:
Match has the same logic.

Reply:
Made all checks matched, also increased error descriptions by a factor of about 2